### DISCUSSION TITLE
CSS can be injected into the page via README.md

### DISCUSSION THREAD

**Discussion by alperenisa** [community/community#127777] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The user here seems to be using the `math` module.
https://github.com/lustryrose882
![resim](https://github.com/community/community/assets/149863334/ff368881-0265-41fb-a731-72bf6ceb00b5)

**Comment by athul-22** [comment]
To confirm the problem, you can try the following steps:

Create a README.md File: Add potentially malicious CSS to a README.md file in a test repository.
markdown
Copy code
# Test

<style>
body {
    background-color: red;
}
</style>
Push to GitHub: Push the file to a GitHub repository and view the rendered README.md on GitHub.
